dist/scripts/shopstandards.js

```diff
@@ -0,0 +1,138 @@
+"use strict";
```

Comment:
I think this file is generated due to a wrong build process. Not sure what the correct command is, or if there's an npm script in the `package.json`. AFAIK this file is generated if the file is concatenated. So this could possibly prevented by `$ gulp build --concat=false` and/or `$ gulp build --concat=false --production`.

Reply:
I've run both and the `--production` flag is needed to avoid generating `shopstandards.js`. I've also added "build" npm script to make it more obvious `npm run build`